streams/src/main/java/org/apache/kafka/streams/state/internals/RecordConverter.java

```diff
@@ -47,5 +49,16 @@ static RecordConverter converter() {
                 record.leaderEpoch()
             );
         };
+
+        // privatize the constructor so the class cannot be instantiated (only used for its static members)
+        private RecordConverters() {}
+
+        public static RecordConverter rawValueToTimestampedValue() {
+            return RAW_TO_TIMESTAMED_INSTANCE;
+        }
+
+        public static RecordConverter identity() {
+            return IDENTITY_INSTANCE;
+        }
```

Comment:
Also added a cached identity function. IMHO, it's just a little more intentional than saying `entity -> entity` inline.

Reply:
nit: `identityRecordConverter` or `noOpRecordConverter` ?